### ISSUE TITLE
[Feature Request] Make the name prompt when duplicating requests and folders optional

### ISSUE THREAD

**Issue by robcowart** [getinsomnia/insomnia#1878]
My Insomnia client auto-updated, and I ran into PR #1839 (Prompt for name when duplicating requests and folders)

For various reasons I will often make dozens of duplicates of a request, which receive minor edits to then kickoff multiple long running requests in parallel. It was very easy to do this in the past by simply right-click -Y duplicate. It didn't matter that it only appended a number the duplicates. in fact that was preferred.

PR #1839 makes it much more of pain to create duplicates. I can understand how some might find the new method convenient, but for me it is exactly the opposite.

Can you make the behaviour a configurable option, giving users the choice?
